DESCRIPTION.html

```diff
@@ -1,3 +1,5 @@
+Probably the sharpest git repository organizer and rebase/merge workflow automation tool you've ever seen!
```

Comment:
**TODO for Marketplace**: I don't think the static image we have there:

![image](https://user-images.githubusercontent.com/3383210/90680649-77a20280-e262-11ea-966b-49d0038d593f.png)

is the most fortunate choice (too many details, yellow edge etc.). My suggestion is to replace it with docs/marketplace/sample_status.png - WDYT :question:

Reply:
sounds good, agreed (sounds like Hagrid :Xd:)

Reply:
Done